torch/jit/_builtins.py

```diff
@@ -55,17 +56,30 @@
     (math.asinh, "aten::asinh"),
     (math.atanh, "aten::atanh"),
     (math.acosh, "aten::acosh"),
-    (math.sinh, "aten::sinh"),
-    (math.cosh, "aten::cosh"),
-    (math.tanh, "aten::tanh"),
     (math.fmod, "aten::fmod"),
     (math.modf, "aten::modf"),
     (math.factorial, "aten::factorial"),
     (math.frexp, "aten::frexp"),
-    (math.isnan, "aten::isnan"),
```

Comment:
Why have the these been deleted?

Reply:
just deleted some duplicate entries!

Reply:
Phew, ok. The existing entries were hidden by the preview.